catalog/ui/search-ui/search-ui-app/pom.xml

```diff
@@ -46,28 +46,27 @@
                     </execution>
                 </executions>
             </plugin>
+            <!-- Puts the features XML file for this app into the maven repo. -->
```

Comment:
Should we move this comment to the next `<plugin/>` element and maybe have a different one here if we feel it'd be helpful?